test/unit/gateways/mercado_pago_test.rb

```diff
@@ -529,6 +529,28 @@ def test_set_binary_mode_to_nil_when_request_is_3ds
     end.respond_with(successful_authorize_response)
   end
 
+  def test_should_not_include_sponsor_id_when_test_mode_is_enabled
+    @options[:sponsor_id] = '1234'
+
+    stub_comms do
+      @gateway.purchase(@amount, @credit_card, @options)
+    end.check_request do |endpoint, data, _headers|
+      assert_not_match(%r("sponsor_id":), data) if endpoint =~ /payments/
```

Comment:
❓ By curiosity.
 Why in this case didn't use this?

if /payments/.match?(endpoint) like in "test_should_include_sponsor_id_when_test_mode_is_disabled"?